src/sagemaker/session.py

```diff
@@ -17,20 +17,24 @@
 
 import os
 import pkg_resources
+import platform
 import sys
 import time
 
 import boto3
-import botocore.session
+import botocore
 import json
 import six
 from botocore.exceptions import ClientError
 
 from sagemaker.utils import name_from_image
 import sagemaker.logs
 
-SDK_VERSION = pkg_resources.require('sagemaker')[0].version
 
+SDK_VERSION = pkg_resources.require('sagemaker')[0].version
+OS_NAME = platform.system()
```

Comment:
If no OS name can be resolved, ``platform.system()`` will return an empty string. Can this be something like:

``OS_NAME = platform.system() or 'UnresolvedOS'``

Same for ``OS_VERSION``

Reply:
done. Also moved everything to a new module.